PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -367,6 +367,11 @@
                 return nil
             }
 
+            // this will bail on interactive animations (e.g edge slide)
+            if isAnimatingTransition(window) {
+                return nil
+            }
```

Comment:
could be just an `or` operation with the condition above.

Reply:
If there's a transition happening, we will skip the screenshot and that is fine, do we capture a screenshot when the transition is over or do we have to wait for a few user interaction that triggers an UI update?
My question is if we are skipping a screenshot somehow, if that is the case, we could "delay" this screenshot if possible